src/Suspense.js

```diff
@@ -0,0 +1,62 @@
+import Brahmos from './index';
+import { Component } from './Component';
+import { forwardRef } from './refs';
+import createContext from './createContext';
+
+const { Provider, Consumer } = createContext();
+
+export class Suspense extends Component {
+    state = {
```

Comment:
we are using buble to bundle Brahmos, buble creates small chunk size than babel. Class property are not supported in buble. Change this to 
```js
constructor() {
   this.state = { resolved: true };
   this.lazyElements = []
}
```

Reply:
Thanks for information. I read in babel how it works.  [Here it is something ](https://babeljs.io/docs/en/babel-plugin-proposal-class-properties) which makes to work. I learnt a new thing. 
I've moved the code to new snippet. But now we started getting a lot of circular dependecy, which we shall look one by one.